workshops/tfx-caip-tf21/lab-01-tfx-walkthrough/lab-01.ipynb

```diff
@@ -71,7 +71,7 @@
     "from tfx.components import BigQueryExampleGen\n",
```

Comment:
<p>When I run this cell in the JupyterLab, I am getting this error</p><p> </p><p><code>`</code>python</p><pre class="ql-syntax" spellcheck="false"> Error importing tfx_bsl_extension.arrow.array_util. Some tfx_bsl functionalities are not availableError importing tfx_bsl_extension.arrow.table_util. Some tfx_bsl functionalities are not availableError importing tfx_bsl_extension.arrow.table_util. Some tfx_bsl functionalities are not available
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
&lt;ipython-input-1-c95642f63133&gt; in &lt;module&gt;
     17 from tfx.components import Evaluator
     18 from tfx.components import ExampleValidator
---&gt; 19 from tfx.components import InfraValidator
     20 from tfx.components import Pusher
     21 from tfx.components import ResolverNode

ImportError: cannot import name 'InfraValidator' from 'tfx.components' (/opt/conda/lib/python3.7/site-packages/tfx/components/__init__.py)
</pre><p> </p><p><code></code>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I am unable to replicate the error you are seeing. Are you using perhaps the Python runtime? I am using the Tensorflow 2.1 Enterprise version; should we prefer one over the other?

Here are the package versions I have in my environment:

Tensorflow Version: 2.1.0
TFX Version: 0.21.4
TFDV Version: 0.21.5
TFMA Version: 0.21.6

Reply:
Added another commit for explicitly listing lab package versions and gave instructions and code to update the packages in the current Jupyter kernel.

Reply:
<p>LGTM</p>